### DISCUSSION TITLE
Load testing thresholds for artillery playwright per page

### DISCUSSION THREAD

**Discussion by jose-JSQ** [artilleryio/artillery#2166] [Artillery]
When I run load test scenario with playwright engine, the Summary report provides all metrics by http page visited.
For instance:
browser.page.FCP.https://artillery.io
browser.page.FCP.https://artillery.io/cloud/
However, when adding thresholds, I am not able to use the whole urls.
for instance:
ensure:
    thresholds:
      - "browser.page.FCP.https://artillery.io/cloud/":1000. (does not work)

Is there a way to use a "name" instead of page, similar to api testing?

**Comment by bernardobridge** [comment]
Hi @jose-JSQ,

Have you enabled the `ensure` plugin in your config like this?
```yaml
  plugins:
    ensure: {}
```

Regardless, it does seem like there's an issue with parsing the URL's like that. In the meantime, you can use this aggregation if you want https://www.artillery.io/docs/reference/engines/playwright#aggregate-metrics-by-scenario-name, and that will aggregate by the scenario name, and then you can use ensure to assert on that.

The disadvantage is that it will group all the requests, so you'll lose the grouping per URL.

---

We'll investigate the bug with `ensure` and whole URL's.
